examples/howto/marginalizing-models.ipynb

```diff
@@ -0,0 +1,1527 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    plt.axvline(x=1891, c="orange")
</pre><p>Compute x from idata to be clear where the number comes from (the posterior mean or mode I assume?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/628/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>